controllers/volumereplicationgroup_controller.go

```diff
@@ -773,16 +775,29 @@ func (v *VRGInstance) reconcileVRsAsPrimary() bool {
 			continue
 		}
 
-		if err := v.processVRAsPrimary(pvcNamespacedName, log); err != nil {
-			log.Info("Requeuing due to failure in getting or creating VolumeReplication resource for PersistentVolumeClaim",
-				"errorValue", err)
+		// Protect the PV backend data in the storage volume as primary.
+		if err := v.protectPVBackendDataAsPrimary(pvcNamespacedName, log); err != nil {
+			requeue = true
+
+			continue
+		}
 
+		// Protect the PV cluster data (in etcd)
+		if err := v.protectPVClusterData(pvc, log); err != nil {
```

Comment:
This method is intended to protect PV Cluster Data (PV resource in the initial release, but, in future, this method will also protect PV snapshot resources).  Specifically, it is not protecting "PVC" cluster data. I agree it will be consistent with other naming, but I was worried that using the term PVC may be misleading that Ramen is protecting PVC cluster data, isn't it?

Reply:
The confusion here is are we abstracting the methods as prepare PVC -> protect PV data -> protect PV cluster data, because of the renaming of `processVRAsPrimary` as `protectPVBackendDataAsPrimary` as we are dropping VR there (the change from process to protect is fine btw).

If the above were renamed from `processVRAsPrimary` to `protectVRAsPrimary`, it automatically qualifies as protecting backend data and acting on a VR. Then this method can be named `protectPVClusterData` without the PVC name. The flow is prepare PVC -> protect VR -> protect PV cluster data hence.

Reply:
> protect VR
The method is `not` protecting the VR though. The method is actually protecting the PV (this is what it does) by creating a VR (this is how it does it).

Reply:
If you want to retain the VR in the name at this level, then I'll either leave the method name as `processVRAsPrimary` or `createVRAsPrimary`.

Reply:
`proess` then, not create as it may create or update